Allow overriding language territory names (#4620)
indico/indico: docs/source/config/settings.rst

    Default: ``{}``

.. data:: CUSTOM_LANGUAGES

    A dict with language/territory name overrides.  This can be useful if the
    official territory name that goes along with a language does not match what
    your Indico instance's target audience expects for a country, e.g. due to
    political situations.

>     For example, to use "Chinese (Simplified)" instead of "Chinese (China)",
>     you would use this

Reviewer: ```suggestion
    For example, to replace "Chinese (Simplified)" with "Chinese (China)",
    you would use:
```
